populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
Why does the rest of the repo need to change? Just move these files into their own module.